cime_config/acme/config_files.xml

```diff
@@ -117,6 +117,23 @@
     <desc>file containing specification of all pe-layouts for primary component (for documentation only - DO NOT EDIT)</desc>
   </entry>
 
+  <entry id="SYSTEM_TESTS_DIR">
```

Comment:
Could we just assume that SRCROOT/components/*/cime_config/SystemTests contain potential system tests? That would allow us to avoid adding another entry.